doc/changelog/1.12.0-changelog.rst

```diff
@@ -225,7 +225,7 @@ A total of 418 pull requests were merged for this release.
 * `#7240 <https://github.com/numpy/numpy/pull/7240>`__: Change 'pubic' to 'public'.
 * `#7241 <https://github.com/numpy/numpy/pull/7241>`__: MAINT: update doc/sphinxext to numpydoc 0.6.0, and fix up some...
 * `#7243 <https://github.com/numpy/numpy/pull/7243>`__: ENH: Adding support to the range keyword for estimation of the...
-* `#7246 <https://github.com/numpy/numpy/pull/7246>`__: DOC: metion writeable keyword in as_strided in release notes
+* `#7246 <https://github.com/numpy/numpy/pull/7246>`__: DOC: mention writeable keyword in as_strided in release notes
 * `#7247 <https://github.com/numpy/numpy/pull/7247>`__: TST: Fail quickly on AppVeyor for superseded PR builds
```

Comment:
These come from PR titles - so arguably we should rename those if we change these. Not really a big deal though.

Reply:
I wouldn't bother with the changelogs, they are what they are.

Reply:
But it is good to remind reviewers that the PR title should be checked and fixed up if needed.

Reply:
Checking and changing PR titles should probably be part of the release process. I've done some of that but maybe it should be formalized a bit more.